jsonrpc/types.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/0xPolygon/minimal/types"
 )
 
+// For union type of transaction and types.Hash
+type transactionOrHash interface {
+	getHash() types.Hash
```

Comment:
Actually this method is not used, but I want to limit types of value to be set.

Reply:
Thank you